run-barr-e-brain.sh

```
@@ -0,0 +1 @@
+./knitware-bots/barr-e/brain/build/barr-e-brain-ubuntu-x64-debug/barr-e-brain
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)